internal/manifest/report.go

```diff
@@ -52,26 +58,48 @@ func (r *Report) Add(slice *setup.Slice, fsEntry *fsutil.Entry) error {
 		return fmt.Errorf("cannot add path to report: %s", err)
 	}
 
+	var hardLinkID uint64
+	sha256 := fsEntry.SHA256
+	size := fsEntry.Size
+	link := fsEntry.Link
```

Comment:
This is a step on the good direction but I think it is less error prone to do `fsEntryCpy := *fsEntry` and then use the fields from the other object.

Reply:
Changes patched. However, I would argue this is useless and complicating the code (also consuming more memory), since the rest of the fields (`Path`, `Mode`, `HardLink`) are never overwritten.